Packs/MicrosoftExchangeOnPremise/Integrations/EWSv2/EWSv2.yml

```
@@ -964,6 +964,13 @@ script:
       description: The contents (body) of the email to send.
     - name: htmlBody
       description: HTML formatted content (body) of the email to be sent. This argument overrides the "body" argument.
+    - name: bodyType
+      description: 'Whether message response body type is text or HTML.'
+      auto: PREDEFINED
+      defaultValue: Text
+      predefined:
+      - HTML
+      - Text
```

Suggestion. `text` is the default value, why not make it 1st?
```suggestion
      - Text
      - HTML
```